### DISCUSSION TITLE
Advanced checklist does not load in mobile app

### DISCUSSION THREAD

**Discussion by coolehs** [standardnotes/forum#2912] [General]
- Uninstalled and reinstalled multiple times, did not fix the issue.
- ![IMG_1594](https://user-images.githubusercontent.com/50015607/204783903-4bae27ed-9a7e-4b35-abf2-6bfe44e1632b.PNG)

**Comment by effieeee** [comment]
Hi @coolehs, this is a known issue at the moment, but I'm afraid the team may not prioritize fixes on this since Advanced Checklist has been deprecated, and is now a [community plugin](https://github.com/standardnotes/plugins) 😞
